webstruct/wapiti.py

```diff
@@ -124,6 +124,10 @@ def merge_top_n(chains):
 
 
 class WapitiCRF(BaseSequenceClassifier):
+    """
+    >>> run_command(['which', WapitiCRF.WAPITI_CMD], verbose=False)
```

Comment:
I think it should be moved to regular tests - WapitiCRF is a public class, its docstring should show its description and usage examples.